src/components/Tree/Tree.js

```diff
@@ -149,7 +150,7 @@ export const Tree = ({
   const nodeList = () => treeModel.current.children.map(child => (
     <TreeItem
       key={`${child.elementHash}`}
-      filterText={filterText}
+      filterText={filterText.trim()}
```

Comment:
Seems odd/bad to run operations on props as we provide them. Can we pull it out of the assignment and do it before we construct the TreeItem or determine why its messing up where you are doing the lowercase? `.trim()` should not remove whitespace inside strings, it should be a combo of `trimStart` and `trimEnd`.